matter_server/server/device_controller.py

```diff
@@ -527,38 +527,27 @@ async def _call_sdk(self, func: Callable[..., _T], *args: Any, **kwargs: Any) ->
 
     async def _check_subscriptions_and_interviews(self) -> None:
         """Run subscriptions (and interviews) for known nodes."""
+        interview_tasks: list[asyncio.Task] = []
+        subscription_tasks: list[asyncio.Task] = []
         for node_id, node in self._nodes.items():
             # (re)interview node (only) if needed
             if (
                 node is None
                 or node.interview_version < SCHEMA_VERSION
                 or (datetime.utcnow() - node.last_interview).days > 30
             ):
-                try:
-                    await self.interview_node(node_id)
-                except NodeInterviewFailed as err:
-                    LOGGER.warning(
-                        "Unable to interview Node %s, we will retry later in the background.",
-                        node_id,
-                        exc_info=err,
-                    )
-                    continue
+                interview_tasks.append(
+                    asyncio.create_task(self.interview_node(node_id))
```

Comment:
We should probably still catch `NodeInterviewFailed` so we need a wrapping function that can do that and that we can schedule as a task.